src/plotman/plot_util.py

```diff
@@ -1,13 +1,14 @@
 import math
 import os
 import re
+import shutil
 
 GB = 1_000_000_000
 
 def df_b(d):
     'Return free space for directory (in bytes)'
-    stat = os.statvfs(d)
-    return stat.f_frsize * stat.f_bavail
+    total, used, free = shutil.disk_usage(d)
+    return free
```

Comment:
This is a bit of a nuanced preference against tuples for returning multiple values.  Yeah, the stdlib is unlikely to change this interface ever but in general I would rather use names to identify values rather than order of elements in a sequence.  And yes, we do use order in plenty of places.  It's one of the many things I want to clean up.

Does this look ok to you?

```suggestion
    usage = shutil.disk_usage(d)
    return usage.free
```

Reply:
Okay, who should commit the change.

Reply:
The source code of shutil.disk_usage() does show that `free` is indeed available space in linux systems.

```
if hasattr(os, 'statvfs'):

    __all__.append('disk_usage')
    _ntuple_diskusage = collections.namedtuple('usage', 'total used free')
    _ntuple_diskusage.total.__doc__ = 'Total space in bytes'
    _ntuple_diskusage.used.__doc__ = 'Used space in bytes'
    _ntuple_diskusage.free.__doc__ = 'Free space in bytes'

    def disk_usage(path):
        """Return disk usage statistics about the given path.

        Returned value is a named tuple with attributes 'total', 'used' and
        'free', which are the amount of total, used and free space, in bytes.
        """
        st = os.statvfs(path)
        free = st.f_bavail * st.f_frsize
        total = st.f_blocks * st.f_frsize
        used = (st.f_blocks - st.f_bfree) * st.f_frsize
        return _ntuple_diskusage(total, used, free)

elif _WINDOWS:

    __all__.append('disk_usage')
    _ntuple_diskusage = collections.namedtuple('usage', 'total used free')

    def disk_usage(path):
        """Return disk usage statistics about the given path.

        Returned values is a named tuple with attributes 'total', 'used' and
        'free', which are the amount of total, used and free space, in bytes.
        """
        total, free = nt._getdiskusage(path)
        used = total - free
        return _ntuple_diskusage(total, used, free)
```

Reply:
Generally, yes, if you like the suggestion you would click the commit button underneath it.  I just went ahead and did it in this case.